### DISCUSSION TITLE
Add tag for Zig

### DISCUSSION THREAD

**Discussion by jlafayette** [BattlesnakeOfficial/feedback#196] [Feature Requests]
Could we add a tag for the Zig programming language?
https://ziglang.org/

**Comment by bvanvugt** [comment]
Definitely, will try to get it added this week.

**Comment by aurorawalker** [comment]
This is done and done! Zig is now an option in the Battlesnake tags.

<img width="503" alt="Screen Shot 2022-05-26 at 9 02 15 AM" src="https://user-images.githubusercontent.com/663009/170527603-b706fdb8-dc8a-4ee4-8514-37178e0a206f.png">

**Comment by jlafayette** [comment]
Yay!  Thank you
